apex-squid/src/test/resources/metrics/comments.cls

```diff
@@ -0,0 +1,20 @@
+/*
+    Some license for this class 
+    * comment with star
+    this license uses "double quotes"
+ 
+ */
+/* another comment*/
+/**/
+//NOSONAR
```

Comment:
/*
asd*/
asd
*/  

Hmm it seems that case is detected at compilation time. maybe it is not necessary to add it but i just test that case in our implementation, it does make test fail, for all comments.